### ISSUE TITLE
Generic Type display problem

### ISSUE THREAD

**Issue by alvindaiyan** [springfox/springfox#1379]
if the return type has generic types, the static html page looks bad:
 SomethingÂ«SomethingImplÂ» 

the «...» is from springofx. Because the api has a generic type with the return. In this case, it's Something<SomethingImpl>. The springfox will generate a json string Something«SomethingImpl» and the codegen will generate "SomethingÂ«SomethingImplÂ»".

**Comment by alvindaiyan** [comment]
@dilipkrish Is this just a workaround?
And yes, I'm currently using ```forCodeGeneration()``` for the generic types

**Comment by dilipkrish** [comment]
No this is a configuration option

**Comment by alvindaiyan** [comment]
This sounds good for me. Feel free to close the issue. 

Thanks

**Comment by sac10nikam** [comment]
Hi Dilip,

I have been trying to create API specification using springfox-swagger-ui, springfox-swagger, and swagger-springmvc. I have spring 4.X with interface drivern REST services.

I have interface which is having REST annotations. In the same file I have added Swagger annotations like @Api, @ApiResponse, @ApiOperation etc along with all details
I have its implementation. But at the moment I guess we are not supporting interfaces as part of swagger project. I have found a way out using RestEasy 2.3.5 but It is also not supporting Spring 4.x.

Using RestEasy we can support interface driven swagger specification but doesn't support spring 4.x annotations. But I would like to do it using your way.

Thanks,
Sachin
